File: Colaveco/JavaModel/web/com/colaveco/gxdomainactiontexts.java
package com.colaveco ;
import com.colaveco.*;
import com.genexus.*;

public final  class gxdomainactiontexts
{
   private static java.util.TreeMap domain = new java.util.TreeMap();
   static
   {
      domain.put(new String((String)"Edit"), "Update");
      domain.put(new String((String)"Delete"), "Delete");
      domain.put(new String((String)"Add"), "Add");
      domain.put(new String((String)"Back"), "Back");
      domain.put(new String((String)"Show filters"), "Show Filters");
      domain.put(new String((String)"Hide filters"), "Hide Filters");
      domain.put(new String((String)"Children"), "Children");
      domain.put(new String((String)"Permission"), "Permission");
      domain.put(new String((String)"Roles"), "Roles");
      domain.put(new String((String)"Login"), "Login");
      domain.put(new String((String)"Register"), "Register");
      domain.put(new String((String)"Logout"), "Logout");
   }

   public static String getDescription( com.genexus.internet.HttpContext httpContext ,
                                        String key )
   {
      if (domain.containsKey( key.trim() ))
      {
         return (String)domain.get( key.trim() );
      }
      else
      {
         return "";
      }
   }

   public static GXSimpleCollection<String> getValues( )
   {
      GXSimpleCollection<String> value = new GXSimpleCollection<String>(String.class, "internal", "");
      java.util.Iterator itr = domain.keySet().iterator();
      while(itr.hasNext())
      {
         value.add((String) itr.next());
      }
      return value;
   }

}

File: Colaveco/JavaModel/mobile/iOS/gxcommon/gxdomainactiontexts.swift
/*
               File: ActionTexts
        Description: ActionTexts
             Author: GeneXus Swift Generator version 17_0_7-156546
       Generated on: November 14, 2022 10:7:19.67
       Program type: Callable routine
          Main DBMS: SQLite
*/
public struct gxdomainactiontexts
{
   private static var domain: [ String : String] =
   [
      "Edit": "Update",
      "Delete": "Delete",
      "Add": "Add",
      "Back": "Back",
      "Show filters": "Show Filters",
      "Hide filters": "Hide Filters",
      "Children": "Children",
      "Permission": "Permission",
      "Roles": "Roles",
      "Login": "Login",
      "Register": "Register",
      "Logout": "Logout",
   ]

   public static func getDescription( _ key: String ) -> String
   {
      if let value = domain[ (key as NSString).gxStringByTrimming() ]
      {
         return value
      }
      else
      {
         return ""
      }
   }

   public static func getValues( ) -> GXObjectCollection
   {
      return GXObjectCollection(array: Array(domain.values))
   }

}

